cmd/e2e/build_unhealthy_gameservers_test.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	mpsv1alpha1 "github.com/playfab/thundernetes/pkg/operator/api/v1alpha1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("Build with unhealthy game servers", func() {
```

Comment:
Could we change the description for "should delete unhealthy gameservers and create new ones" or something like that?

Reply:
will do, thanks